scrabble.js

```diff
@@ -1,14 +1,140 @@
 const Scrabble = {
-  score: function(word) {
-    // TODO: implement score
-  }
+  // https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/10-JavaScript/fun-with-functions.md
+  // in the "attaching functions to objects" what's written is inconsistent
+  // with what's here which is inconsistent with what
+  // linter underlines in red.
+
+  // Documentation says it should be highestScoreFrom: function(params){} but
+  // this makes linter angry
```

Comment:
Good observation. This is our first time running our JavaScript curriculum with full ES6 support, and there are definitely some places where things don't quite line up. What you've written here (without the colon or the `function` keyword) is the correct style going forward, and we will update the textbook to match.